svc-aggregation/agmodel/system.go

```diff
@@ -160,6 +160,21 @@ func GetPluginData(pluginID string) (Plugin, *errors.Error) {
 	return plugin, nil
 }
 
+func GetPluginMgrAddr(pluginID string) (Plugin, *errors.Error) {
```

Comment:
why this function added in agmodel code with random port, id, and host this not ut why it is added

Reply:
yes i agree, we cannot use dummy values in agmodel.

Reply:
Changed; hardcoding removed & tested.